person.rb

```diff
@@ -19,3 +26,45 @@ def of_age?
     @age >= 18
   end
 end
+# capitalizeDecorator.rb
+
+class CapitalizeDecorator < Nameable
+  def initialize(nameable)
+    @nameable = nameable
+    super()
+  end
+
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
+
+class Decorator < Nameable
+  def initialize(nameable)
+    @nameable = nameable
+    super()
+  end
+
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
+
+class TrimmerDecorator < Nameable
+  def initialize(nameable, max_length = 10)
+    @nameable = nameable
+    @max_length = max_length
+    super()
+  end
+
+  def correct_name
+    @nameable.correct_name.slice(0, @max_length)
+  end
+end
```

Comment:
**OPTIONAL** - Good job done so far, but in order to make your code cleaner and easier to read, I suggest you put all Decorators in their individual files. It'd help make your code more modular, please fix this 🙏